### ISSUE TITLE
all List instances deletion is not working in operational db 

### ISSUE THREAD

**Issue by iGovindY** [sysrepo/sysrepo#2544]
Hi 

with new sr_oper_delete_item_str API all instances deletion is not working.
though on running db deletion works

gives below error, here aggregation is a LIST.

| ERROR | SR | sr_delete_item_example | Predicate missing for list "aggregation" in path "/o-ran-aggregation-base:aggregated-o-ru/aggregation". |

```
[root@pwredg85021 ~]# sysrepoctl --version
sysrepoctl - sysrepo YANG schema manipulation tool, compiled with libsysrepo v2.0.29 (SO v6.2.2)

[root@pwredg85021 ~]# yanglint --version
yanglint 2.0.71
```

**Comment by michalvasko** [comment]
Yes, that is a feature, not a bug. Since the stored changes of the operational DS are applied on-demand many times, it is impossible to report changes (if there are some subscriptions) on such generic edits such as deleting all list instances. You must delete specific instances so that the deletion of these instances can be reported for the subscribed callbacks.
